### ISSUE TITLE
getting error on 'make build'

### ISSUE THREAD

**Issue by Misaghlb** [atahani/golang-rest-api-sample#2]
```
# github.com/atahani/golang-rest-api-sample/controller/user
controller/user/user_controller.go:337: invalid operation: token.Claims["exp"] (type jwt.Claims does not support indexing)
controller/user/user_controller.go:338: invalid operation: token.Claims["uid"] (type jwt.Claims does not support indexing)
controller/user/user_controller.go:339: invalid operation: token.Claims["roles"] (type jwt.Claims does not support indexing)
controller/user/user_controller.go:340: invalid operation: token.Claims["name"] (type jwt.Claims does not support indexing)
controller/user/user_controller.go:341: invalid operation: token.Claims["img"] (type jwt.Claims does not support indexing)
controller/user/user_controller.go:342: invalid operation: token.Claims["aid"] (type jwt.Claims does not support indexing)
controller/user/user_controller.go:343: invalid operation: token.Claims["tid"] (type jwt.Claims does not support indexing)
Makefile:29: recipe for target 'build' failed
make: *** [build] Error 2

```

**Comment by Misaghlb** [comment]
solved by using jwt-go version under v3.0.0

**Comment by atahani** [comment]
hi @Misaghlb 

i used [glide](https://github.com/Masterminds/glide) as package management, when you import package to project as dependency `glide` store `HEAD` hash of package repository as version in `glide.lock` so when you run `glide install` command, glide get all of the dependencies in compatible version and locate inside `./vendor/` directory.

i think you have different version of [github.com/dgrijalva/jwt-go](https://github.com/dgrijalva/jwt-go) package in your `$GOPATH` and compile with incompatible version.

![screen shot 2016-08-03 at 12 05 37 am](https://cloud.githubusercontent.com/assets/1076900/17342339/0fb724fc-590e-11e6-8d44-690c441e2b01.png)

i will update whole sample project at this weekend 😊

**Comment by Misaghlb** [comment]
yes i solved with by jwt-go version 2.7 and build it 
but i think its not working correctly so im waiting for your updates :wink:

**Comment by Misaghlb** [comment]
by replacing this for claims the build problem will gone (jwt-go latest ver) :
`token.Claims = jwt.MapClaims{
        "exp": accessToken.ExpireAt.Unix(),
        "uid": u.Id.Hex(),
        "iat": time.Now().Unix(),
        "roles": u.Roles,
        "name": u.DisplayName,
        "img": u.ImageFileName,
        "aid": trustedAppId.Hex(),
        "tid": accessToken.Id.Hex(),
    }`
